include/countly/crash_module.hpp

```diff
@@ -28,7 +28,8 @@ class CrashModule {
    * @param title: a string that contains description of the exception.
    * @param stackTrace: a string that describes the contents of the call-stack.
    * @param segmentation: custom key/values to be reported
-   * @param crashMetrics: contains crash information e.g app version, OS etc...
+   * @param crashMetrics: contains crash information e.g app version, OS.
+   * In crash metrics all values except '_os', '_app_version' and '_error' are optional.
```

Comment:
say it the other way around. say which things "are mandatory" and not which "are not optional"